src/tests/jsonLogicFixtures.js

```diff
@@ -108,6 +108,77 @@ export const schemaWithMissingComputedValue = {
   required: [],
 };
 
+export const schemaWithVarThatDoesNotExist = {
+  properties: {
+    field_a: {
+      type: 'number',
+    },
+  },
+  'x-jsf-logic': {
+    validations: {
+      a_greater_than_ten: {
+        errorMessage: 'Must be greater than 10',
+        rule: {
+          '>': [{ var: 'field_b' }, 10],
+        },
+      },
+    },
+  },
+  required: [],
+};
+
+export const schemaWithDeepVarThatDoesNotExist = {
+  properties: {
+    field_a: {
+      type: 'number',
+    },
+  },
+  'x-jsf-logic': {
+    validations: {
+      a_greater_than_ten: {
```

Comment:
nit-bug: not really greater than 10 anymore 😅 . To make it more generic you could just rename it to "field_a_dummy" with a dummy error message and then it's easier to copy-paste around 😅

Reply:
fixed :)